Incompatible with PostgreSQL 16.0?

With PostgreSQL 16.0 we started seeing the following in our CI, which was using the `postgres:latest` docker image:

```
[...}
15:29:12.091 [info] alter table carbonite_default.triggers

15:29:12.092 [info] create table carbonite_default.outboxes
** (Postgrex.Error) ERROR 22P02 (invalid_text_representation) invalid input syntax for type xid8: "0::TEXT::xid8"
    (ecto_sql 3.10.2) lib/ecto/adapters/sql.ex:1047: Ecto.Adapters.SQL.raise_sql_call_error/1
    (elixir 1.15.5) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
    (ecto_sql 3.10.2) lib/ecto/adapters/sql.ex:1154: Ecto.Adapters.SQL.execute_ddl/4
    (ecto_sql 3.10.2) lib/ecto/migration/runner.ex:327: Ecto.Migration.Runner.log_and_execute_ddl/3
    (elixir 1.15.5) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
    (elixir 1.15.5) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
    (ecto_sql 3.10.2) lib/ecto/migration/runner.ex:290: Ecto.Migration.Runner.perform_operation/3
    (stdlib 5.0.2) timer.erl:270: :timer.tc/2
Error: Process completed with exit code 1.
```

Have you verified that Carbonite is compatible with PG 16.0? We've specified `postgres:15.4` in our CI so not an issue for us for now, FYI.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,9 @@
+## Unreleased
+
+### Fixed
+
+* Fixed a minor incompatibility with recently released Postgres 16.0.
+
 ## [0.9.0] - 2024-04-27
 
 **New migration patches:** 6

--- a/lib/carbonite/migrations/v3.ex
+++ b/lib/carbonite/migrations/v3.ex
@@ -18,7 +18,7 @@ defmodule Carbonite.Migrations.V3 do
     create table("outboxes", primary_key: false, prefix: prefix) do
       add(:name, :string, null: false, primary_key: true)
       add(:memo, :map, null: false, default: "{}")
-      add(:last_transaction_id, :xid8, null: false, default: "0::TEXT::xid8")
+      add(:last_transaction_id, :xid8, null: false, default: "0")
 
       timestamps(type: :utc_datetime_usec)
     end

